lms/templates/courseware/progress_graph.js

```diff
@@ -1,7 +1,11 @@
 <%page args="grade_summary, grade_cutoffs, graph_div_id, show_grade_breakdown = True, show_grade_cutoffs = True, **kwargs"/>
 <%!
-import json
-import math
+    import json
+    import math
+    import bleach
```

Comment:
nit: does bleach start a new section?  if so, add an empty line.  if not, put in alphabetical order.